README.md

```diff
@@ -84,6 +84,7 @@
       - [Fetching Group Details](#fetching-group-details)
       - [Creating and Updating a Group](#creating-and-updating-a-group)
       - [Deleting a Group](#deleting-a-group)
+      - [Executing Full System Export](#executing-a-full-system-export)
```

Comment:
Executing Full System Export
-->
Generating Full System Export

executing-a-full-system-export
-->
generating-full-system-export

(we should also change the titles outside of the table of contents).